doc/secure_boot_guide.md

```diff
@@ -55,8 +55,8 @@ Using ECDSA NIST P384 as example:
 
 Clear environment varibles CC and AR at first:
 ```
-set CC=
-set AR=
+set CC=clang

```

Comment:
The comment above says about "clear", the code is actually set, so make them consistent?

Reply:
Yes, I have corrected it. Thank you.